tests/sqllogictests/suites/temp_table/temp_memory_tables.sql

```diff
@@ -28,7 +28,7 @@ create temp table t2(a int,b int)  engine=MEMORY;
 statement error 1005
 create temp table t2(a INT auto_increment) engine=MEMORY;
 
-statement error 1105
+statement error 1006
```

Comment:
why is the ErrorCode changed? (just curious)

Reply:
the format is `statement error <regex>`.
the output of mysql handler also contains an error code of mysql proto.